extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -148,27 +148,27 @@ public function processSharing( $articleID, $sharerID, $IP ) {
 		$this->processCountersForInTrack();
 		$this->saveBadges();
 
-		if ( count( $this->mNewBadges ) > 0 ) {
+		if( count( $this->mNewBadges ) > 0 ) {
 			$this->calculateAndSaveScore();
 		}
 
 		wfProfileOut( __METHOD__ );
 	}
 
-	public function processSaveComplete($article, $user, $revision, $status ) {
+	public function processSaveComplete( $article, $user, $revision, $status ) {
 		wfProfileIn( __METHOD__ );
 
 		$this->mUser = $user;
 
-		if ( self::canEarnBadges( $this->mUser ) ) {
+		if( self::canEarnBadges( $this->mUser ) ) {
 
 			$this->mArticle = $article;
 			$this->mRevision = $revision;
 
-			if ( $this->mArticle ) {
+			if( $this->mArticle ) {
 
 				// logic should be processed only one time during one request
-				if ( !self::$mDone ) {
+				if( !self::$mDone ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1066)